src/client.rs

```diff
@@ -193,6 +193,22 @@ impl<'a> Client<'a> {
         .await
     }
 
+    /// Get health of MeiliSearch server, return true or false.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use meilisearch_sdk::{client::*};
+    /// #
+    /// # futures::executor::block_on(async move {
+    /// let client = Client::new("http://localhost:7700", "masterKey");
+    /// let health = client.is_healthy().await;
+    /// # });
+    /// ```
+    pub async fn is_healthy(&self) -> bool {
+        self.health().await.is_ok()
+    }
```

Comment:
```suggestion
    pub async fn is_healthy(&self) -> bool {
        if let Some(health) = self.health().await {
            health.status.as_str() == "available"
        } else {
            false
        }
    }
```
There is a mistake here. Calling `is_ok` on the Result will return whether the HTTP request was successful. So it would return `true` almost always, even when `status` is not `available`. I think you wanted to check the content of `status`. That can be done this way.

Reply:
@Mubelotix  This is exactly what I want to do. Thanks for your help!